google/cloud/monitoring_v3/query.py

```diff
@@ -481,9 +486,9 @@ def _build_query_params(self, headers_only=False, page_size=None):
             )
 
         if headers_only:
-            params["view"] = enums.ListTimeSeriesRequest.TimeSeriesView.HEADERS
+            params["view"] = monitoring_v3.ListTimeSeriesRequest.TimeSeriesView.HEADERS
         else:
-            params["view"] = enums.ListTimeSeriesRequest.TimeSeriesView.FULL
+            params["view"] = monitoring_v3.ListTimeSeriesRequest.TimeSeriesView.FULL
```

Comment:
Outrageous nit, feel free to ignore: while we're changing things, why not simplify this a little?
```python
tsv = monitoring_v3.ListTimeSeriesRequest.TimeSeriesView
params["view"] = tsv.HEADERS if headers_only else tsv.FULL
```

Reply:
done